pom.xml

```diff
@@ -22,7 +22,7 @@
   <parent>
     <groupId>org.apache.maven.shared</groupId>
     <artifactId>maven-shared-components</artifactId>
-    <version>34</version>
+    <version>36</version>
```

Comment:
maybe separate PR and issue

Reply:
I don't think it would have value, first i updated java and then updated pom. In pom.xml we have almost nothing